src/main/resources/assets/openblocks/lang/de_de.lang

```diff
@@ -15,63 +15,63 @@ enchantment.openblocks.laststand=Letzte Rettung
 enchantment.openblocks.flimflam=Blödsinn
 
 openblocks.gui.blocks=Blöcke
-openblocks.gui.items=Items
+openblocks.gui.items=Gegenstände
 openblocks.gui.misc=Verschiedenes
-openblocks.gui.changelogs=Changelogs
+openblocks.gui.changelogs=Änderungsprotokoll
 
-openblocks.gui.features=Features
-openblocks.gui.bugfixes=Bugfixes
-openblocks.gui.tweaks=Tweaks
+openblocks.gui.features=Eigenschaften
+openblocks.gui.bugfixes=Fehlerbehebung
+openblocks.gui.tweaks=Optimierungen
 
 openblocks.gui.luggage=Reisegepäck
 openblocks.gui.sprinkler=Rasensprenger
 openblocks.gui.vacuumhopper=Vakuumtrichter
 openblocks.gui.bigbutton=Großer Knopf
 openblocks.gui.xpbottler=XP-Abfüllgerät
-openblocks.gui.autoanvil=Auto-Amboss
-openblocks.gui.autoeject=Auto-Auswerfen
-openblocks.gui.autoextract=Auto-Extrahieren
-openblocks.gui.autodrink=Auto-Trinken
-openblocks.gui.autoenchantmenttable=Auto-Zaubertisch
+openblocks.gui.autoanvil=Automatischer Amboss
+openblocks.gui.autoeject=Automatischer Auswerfer
```

Comment:
This... doesn't fit in the GUI.

Reply:
"Autom." is the solution. "Auto" also means "car" in german. :D

Reply:
Yes, short for "Automobil", same meaning as "Auto" here.